sources/packages/backend/apps/db-migrations/src/sql/SFASRestrictionMaps/Create-sfas-restriction-maps.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE sims.sfas_restriction_maps (
+  id SERIAL PRIMARY KEY,
+  legacy_code VARCHAR(4) NOT NULL,
```

Comment:
Asking for my understanding. Are we certain that legacy code will not be more than 4 characters?

Reply:
The values follow the source/destination information which is, `sims.sfas_restrictions.code` column is set to 4, and `sims.restrictions.restriction_code` is set to 10.